actors/src/main/kotlin/ktx/actors/actions.kt

```diff
@@ -33,65 +33,77 @@ operator fun Actor.plusAssign(action: Action) = addAction(action)
  * @return [SequenceAction] storing both actions.
  */
-infix fun Action.then(action: Action): SequenceAction {
-  val result = Actions.sequence()
-  result.addUnwrapped(this)
-  result.addUnwrapped(action)
-  return result
-}
+infix fun Action.then(action: Action): SequenceAction = Actions.sequence(this, action)
 
-private fun SequenceAction.addUnwrapped(action: Action) {
-  when (action) {
-    is SequenceAction -> action.actions.forEach(::addAction)
-    else -> addAction(action)
-  }
+/**
+ * Adds [action] to this [SequenceAction].
+ *
+ * @param action will be added to this [SequenceAction]
+ * @return [SequenceAction] this [SequenceAction]
+ */
+infix fun SequenceAction.then(action: Action): SequenceAction {
+  addAction(action)
+  return this
 }
 
 /**
- * Wraps this action and the passed action with a [SequenceAction].
- * @param action will be executed after this action.
+ * Wraps this action and the passed action in a [SequenceAction].
+ *
+ * @param action will be executed after [this] action.
  * @return [SequenceAction] storing both actions.
  */
 operator fun Action.plus(action: Action): SequenceAction = then(action)
 
 /**
- * Adds another [Action] to this sequence.
+ * Adds [action] to this [SequenceAction].
+ *
  * @param action will be executed during this sequence.
  */
 operator fun SequenceAction.plusAssign(action: Action) = addAction(action)
 
 /**
- * Actions utility. Wraps this action and the passed action with a [ParallelAction], executing them both
- * at the same time.
- *
- * If [this] or [action] are [ParallelAction]s, their nested actions will be unwrapped.
+ * Wraps this action and the passed action with a [ParallelAction], executing them both at the same time.
  *
  * @param action will be executed at the same time as this action.
  * @return [ParallelAction] storing both actions.
  */
-infix fun Action.parallelTo(action: Action): ParallelAction {
-  val parallel = Actions.parallel()
-  parallel.addUnwrapped(this)
-  parallel.addUnwrapped(action)
-  return parallel
-}
+infix fun Action.along(action: Action): ParallelAction = Actions.parallel(this, action)
+
+@Deprecated("parallelTo has been replaced with along.", ReplaceWith("this along action"), DeprecationLevel.WARNING)
+infix fun Action.parallelTo(action: Action): ParallelAction = along(action)
 
-private fun ParallelAction.addUnwrapped(action: Action) {
-  when (action) {
-    is ParallelAction -> action.actions.forEach(::addAction)
-    else -> addAction(action)
-  }
+/**
+ * Adds [action] to this [ParallelAction], so long as this isn't a [SequenceAction]. If it is a [SequenceAction],
+ * [@see Action.along].
+ *
+ * @param action will be added to this [ParallelAction].
+ * @return [ParallelAction] this [ParallelAction], now containing [action]]
+ */
+infix fun ParallelAction.along(action: Action): ParallelAction {
+  if (this is SequenceAction)
+    return this as Action along action // workaround for SequenceAction inheritance from ParallelAction
```

Comment:
I think this would be cleaner:

```kotlin
if (this is SequenceAction) {
  // Workaround for SequenceAction inheritance from ParallelAction:
  return Actions.parallel(this, action)
}
```